entries/c/cornell.edu.json

```diff
@@ -8,6 +8,9 @@
       "u2f",
       "call"
     ],
+    "custom-software": [
+      "Duo"
+    ],
```

Comment:
Maybe remove the custom-hardware as u2f is used?

Reply:
Looks like D100 tokens are supported as well https://it.cornell.edu/twostep/choose-your-authentication-device-two-step-login